multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/helpers/SystemParametersTest.java

```diff
@@ -84,14 +84,14 @@ void testDescriptorOverridesDefaults() throws Exception {
 
     static Stream<Arguments> testModuleParameters() {
         return Stream.of(
-                         // [1] Do not reserve temporary routes and does not apply namespace
-                         Arguments.of(false, false),
-                         // [2] Reserve temporary routes and does not apply namespace
-                         Arguments.of(true, false),
-                         // [3] Do not reserve temporary routes but use namespace
-                         Arguments.of(false, true),
-                         // [4] Reserve temporary routes and apply namespace
-                         Arguments.of(true, true));
+            // [1] Do not reserve temporary routes and does not apply namespace
+            Arguments.of(false, false),
```

Comment:
there's no way this was reaching the line length limit - something weird is going on in this case to not align the newlines to the opening bracket